exercises/sum-of-multiples/example.rb

```diff
@@ -1,6 +1,6 @@
 class SumOfMultiples
   def self.to(limit)
```

Comment:
Seems like it hasn't been used for a while.  I don't remember removing the test for this.  Done deal!